src/Documentation/Markdown/Markdown.js

```diff
@@ -2,13 +2,13 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import NextLink from 'next/link'
 import Router from 'next/router'
-// components
+
 import ReactMarkdown from 'react-markdown'
 import { LightButton } from '../LightButton'
 import Tutorials from '../Tutorials'
 import Tooltip from '../../Tooltip'
 import Collapsible from 'react-collapsible'
-// syntax highlighter
```

Comment:
same here - let's not do this right now. Let's postpone cosmetic JS changes please and wait for @iAdramelk to wrap up the two major branches.

Reply:
You mean not to remove those unnecessary comments? I thought we agreed to remove them. Should I bring them back? Up to you, but I think a simple removal like this is very unlikely to produce merge conflicts.

Reply:
yes, I mean not do a lot of cosmetic changes to JS at this moment. We can do it later. It might create conflicts for @iAdramelk for no good reason.

Reply:
OK... Reverted https://github.com/iterative/dvc.org/compare/4950d25..7ea2edb. This is the 2nd time these changes get reverted so I'll just forget about this until/if someone else brings it up 😋